broker-core/src/main/java/io/zeebe/broker/job/JobStateController.java

```diff
@@ -113,18 +112,14 @@ public void create(final long key, final TypedRecord<JobRecord> record) {
     }
   }
 
-  public void activate(final TypedRecord<JobRecord> record) {
-    activate(record.getKey(), record.getValue());
-  }
-
   public void activate(final long key, final JobRecord record) {
     final DirectBuffer type = record.getType();
     final long deadline = record.getDeadline();
 
     DirectBuffer keyBuffer;
     DirectBuffer valueBuffer;
 
-    try (WriteOptions options = new WriteOptions().setSync(true);
```

Comment:
What does this?

Reply:
I'm not sure why this was there in the first place; `setSync(true)` means you flush the write buffers immediately, which I guess is to ensure that even if the OS crashes right after your operation it was still written. We don't need this here

Reply:
Ok thanks.